src/main/java/org/spongepowered/api/text/channel/MessageChannel.java

```diff
@@ -0,0 +1,181 @@
+     */
+    static MessageChannel combined(MessageChannel... channels) {
+        return new CombinedMessageChannel(channels);
+    }
+
+    /**
+     * Gets a message channel that targets all subjects contained within the given channels
+     * and applies the message transformations of each channel in order.
+     *
+     * @param channels The channels to combine
+     * @return The channel
+     * @see CombinedMessageChannel
+     */
+    static MessageChannel combined(Collection<MessageChannel> channels) {
+        return new CombinedMessageChannel(channels);
+    }
+
+    /**
+     * Gets a message channel that targets the given sources.
+     *
+     * @param recipients The recipients
+     * @return The channel
+     * @see FixedMessageChannel
+     */
+    static MessageChannel fixed(MessageReceiver... recipients) {
+        return new FixedMessageChannel(recipients);
+    }
+
+    /**
+     * Gets a message channel that targets the given recipients.
+     *
+     * @param recipients The recipients
+     * @return The channel
+     * @see FixedMessageChannel
+     */
+    static MessageChannel fixed(Collection<? extends MessageReceiver> recipients) {
+        return new FixedMessageChannel(recipients);
+    }
+
+    /**
+     * Broadcast a message to this channel, transforming and sending it to
+     * the members.
+     *
+     * @param original The original text to send
+     */
+    default void broadcast(Text original) {
+        checkNotNull(original, "original text");
+        for (MessageReceiver member : this.getMembers()) {
+            this.transformMessage(null, member, original).ifPresent(member::sendMessage);
+        }
+    }
+
+    /**
+     * Send a message to this channel, transforming and sending it to the members.
+     *
+     * @param sender The sender of the message
+     * @param original The original message to send
+     */
+    default void send(Object sender, Text original) {
+        checkNotNull(sender, "sender");
+        checkNotNull(original, "original text");
+        for (MessageReceiver member : this.getMembers()) {
+            this.transformMessage(sender, member, original).ifPresent(member::sendMessage);
+        }
+    }
+
+    /**
+     * Handle transforming the input message appropriately.
+     * Transforms
+     *
+     * @param sender The sender of the message
+     * @param recipient The recipient of the message
+     * @param original The original message, to optionally transform
+     * @return The message to send, if present, otherwise {@link Optional#empty()}
+     */
+    default Optional<Text> transformMessage(@Nullable Object sender, MessageReceiver recipient, Text original) {
+        return Optional.of(original);
+    }
+
+    /**
+     * Gets a collection of all members in this channel.
+     *
+     * @return A collection of all members of this channel
+     */
+    Collection<MessageReceiver> getMembers();
+
+    /**
+     * Gets a mutable delegate of this channel.
```

Comment:
Should be something more like this:

> Gets or creates a mutable delegate of this channel. if this is already a mutable message channel than this instance is returned. Otherwise the current member list will be copied and used in the newly created message channel which delegates its #transformMessage(...) method to this instance. After that any changes to the member list of this instance won't reflect on the created instance.